frontend/src/context/AuthContext.tsx

```diff
@@ -34,12 +34,13 @@ export const AuthProvider: React.FC<{ children: React.ReactNode }> = ({ children
         verifyAuthToken();
     }, [token]);
 
-    const login = (username: string, email: string, newToken: string, id: string) => {
+    const login = (username: string, email: string, newToken: string, id: string, profileImage: string,) => {
```

Comment:
```suggestion
    const login = (username: string, email: string, newToken: string, id: string, profileImage: string) => {
```

Small nit - extra comma